db/migrate/20221019174418_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,8 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :author, foreign_key: { to_table: :users }
+    add_reference :likes, :post, foreign_key: {to_table: :posts}
+    add_reference :comments, :author, foreign_key: { to_table: :users }
+    add_reference :comments, :post, foreign_key: {to_table: :posts}
+  end
+end
```

Comment:
- [ ] I was unable to set up your app locally because this migration already exists. Kindly remove this file to fix that.